jenkins.yml

```diff
@@ -22,6 +22,22 @@
     sudo_defaults:
       - defaults: '!requiretty'
 
+  pre_tasks:
+    - name: Add jenkins service unit to systemd in case of docker.
+      copy:
+        dest: /etc/systemd/system/jenkins.service
+        content: |
```

Comment:
The best way is to replace scripts from init.d to systemd in RPM package, I suppose (now CentOS, Fedora and RHEL moves systemd from init.d), not Jenkins role in ansible. 
I suppose that we move it to templates at present, adding issue tickets for it to track.

Is that OK?